Equinox/SimpleTimer.h

```diff
@@ -18,7 +18,7 @@ class SimpleTimer
 		runTicks = SDL_GetTicks();
 	}
 
-	int Stop()
+	unsigned int Stop()
```

Comment:
Stop is not working as expected, it must return the time when it was stopped, even if the timer is stopped. Bassically you just need to stop the timer and return Read();